config/default.json

```diff
@@ -1,5 +1,6 @@
 {
 	"host" : "homebrewery.local.naturalcrit.com:8000",
 	"naturalcrit_url" : "local.naturalcrit.com:8010",
-	"secret" : "secret"
-}
\ No newline at end of file
+	"secret" : "secret",
+	"web_port" : 8001,
```

Comment:
Last things, and then I'll merge this:
1) Remove the comma after 8001, it crashes the build step
2) Is there a reason you need the web_port different than the default 8000 we already set? I don't see 8001 used anywhere else in this PR. I'm ok leaving the `web_port` key in here for people who want to easily customize the port, but I would prefer the default value in this file stay 8000, because otherwise it messes with my current development setup.

Reply:
8001 was only used to ensure that the value was actually being used; it can be reverted to 8000. I'll push a commit in a few minutes with that change and the fix for the hanging comma in the config file.

Reply:
I've also changed the port number in the FreeBSD service file so that it will set the environment variable to run the service on port 8000.